neon-animated-pages.html

```diff
@@ -212,6 +216,23 @@
         return element == selectedPage;
       }
       this.notifyResize();
+    },
+
+   /**
+     * Returns distributed nodes either from the slot or
+     * content (dual-mode support).
+     * @private
+     */
+    _getDistributedNodes: function(projectionPoint) {
```

Comment:
Polymer 2.0 implements the method `getDistributedNodes`, so this method is not necessary as you can keep doing `Polymer.dom(this.$.content).getDistributedNodes()`